### PR TITLE
docs:  add credential doc redirects and fix session redirect

### PR DESCRIPTION
**Pull request by randallmorey**
This small PR introduces documentation redirects for credential stores and sources (libraries) intended for use by Boundary product UIs.  And it resolves an inconsistency with the redirect for sessions documentation.

### CODE REVIEW

File: website/redirects.js [hunk 1]
```diff
@@ -100,7 +100,17 @@ module.exports = [
...
+    destination: '/docs/concepts/domain-model/sessions',
+    permanent: false,
+  },
+  {
+    source: '/help/admin-ui/credential-stores',
+    destination: '/docs/concepts/domain-model/credential-stores',
+    permanent: false,
+  },
+  {
+    source: '/help/admin-ui/credential-sources',
```
Comment (gsusmi) [inline website/redirects.js:112]: Is this an early pivot from credential-libraries to credential-sources?

Comment (randallmorey) [reply website/redirects.js:112]: Yeah.  It's just the redirect URL which users will almost never see.

Comment (gsusmi) [reply website/redirects.js:112]: Gotcha. Just checking.

**Review by gsusmi — DISMISSED** [review]
LGTM. Ty!

**Review by calcaide — APPROVED** [review]
LGTM!

**Review by calcaide — APPROVED** [review]
LGTM

**Review by jefferai — APPROVED** [review]

**Pull request merged**